src/components/molecules/pokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import Axios from 'axios';
```

Comment:
Ik heb dit meer mensen zien doen, maar `axios` wordt gewoonlijk met een kleine letter gebruikt (zie [documentatie](https://www.npmjs.com/package/axios)) omdat hoofdletters vaak componenten of andere dingen aanduiden. Dit zal ongetwijfeld werken, maarrrrr ik stem voor kleine letters